plugins/openai_plugin.py

```diff
@@ -100,7 +104,8 @@ def request_http(self, query: dict, user_id: int, test_end_time: float = 0):
             error = message.get("error")
             if error is None:
                 if "/v1/chat/completions" in self.host:
-                    result.output_text = message["choices"][0]['delta']['content']
+                    #result.output_text = message["choices"][0]['delta']['content']
+                    result.output_text = message["choices"][0]['message']['content']
```

Comment:
@ccamacho, I haven't used the chat completions API, but from https://platform.openai.com/docs/api-reference/chat/create, it seems like `message["choices"][0]['message']` is correct, did you see the delta key used by some runtime?

Reply:
Nevermind, I understand now! Delta is used for streaming chunks, message is used for non-streaming. Thanks for the fix!
https://platform.openai.com/docs/api-reference/chat/streaming

Reply:
Good catch! I tested streaming before.